gubernator/main.py

```diff
@@ -34,11 +34,8 @@
 import filters
 
 BUCKET_WHITELIST = {
-    'kubernetes-jenkins',
-    'kube_azure_log',
-    'rktnetes-jenkins',
-    'kubernetes-github-redhat',
-    'kopeio-kubernetes-e2e',
+    re.match(r'gs://([^/]+)', path).group(1)
```

Comment:
Rather than trimming off the `gs://` we could just remove that from `buckets.json` and add it in the test-history maker as needed.